app/src/org/commcare/dalvik/dialogs/PaneledChoiceDialog.java

```diff
@@ -89,4 +90,30 @@ public void setOnDismissListener(DialogInterface.OnDismissListener listener) {
     public void dismiss() {
         dialog.dismiss();
     }
+
+    public void addCollapsibleInfoPane(String messageContent) {
```

Comment:
I love this being part of the framework! Can definitelyimagine this being super helpful.